en/mavgen_c/README.md

```diff
@@ -57,11 +74,7 @@ mavlink/common/../mavlink_helpers.h:86:24: warning: taking address of packed mem
    86 |  crc_accumulate_buffer(&msg->checksum, _MAV_PAYLOAD(msg), msg->len);
 ```
 
-These can be ignored because MAVLink re-orders packed structures such that values are properly aligned.
-Specifically all 4-byte values are aligned on 4-byte boundaries (by putting them first), all 2-byte values come after those and are hence also aligned, and last of all come the 
-the 1-byte values.
-
-You can supress the warnings in CMake using `target_compile_options(mavlink_c INTERFACE -Wno-address-of-packed-member -Wno-cast-align)`.
+You can supress the warnings using `-Wno-address-of-packed-member`.
```

Comment:
1. I kind of like the "reason" why it is OK to suppress warnings - otherwise I'm asking "is this a concern". Because I want to merge this I'll add as a comment and we can return it if you think this is a valid point.
2.  This is simpler, but might have been even better do tod something like
     > You can suppress the warnings using `-Wno-address-of-packed-member`, as shown below
       ```
       target_compile ...`.
       ```
3. Fix typo and add the original info as a comment for easy restoration, if desired.
       
```suggestion
<!--
These can be ignored because MAVLink re-orders packed structures such that values are properly aligned. Specifically all 4-byte values are aligned on 4-byte boundaries (by putting them first), all 2-byte values come after those and are hence also aligned, and last of all come the the 1-byte values.

-->
You can suppress the warnings using `-Wno-address-of-packed-member`.

```